setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
Thanks @rgommers. Did some more tweaking this morning and got RTD to work. 🎉

Yeah the `gcc` name situation is strange. It seems PyWavelets does pick up the funky compiler name correctly, but pyFFTW didn't. Anyways ended up adding a hack that worked around this issue.

Splitting out the docs does still seem like a good idea, but it didn't fix the issue. Seems there were some Sphinx configuration issues that were the cause of the problem. Fixed them in PR ( https://github.com/pyFFTW/pyFFTW/pull/164 ).